src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -832,7 +838,7 @@ class SnowflakeOCSP:
 
     OCSP_WHITELIST = re.compile(
         r"^"
-        r"(.*\.snowflakecomputing\.com$"
+        r"(.*\.snowflakecomputing\.[a-zA-Z]{1,63}$"
         r"|(?:|.*\.)s3.*\.amazonaws\.com$"  # start with s3 or .s3 in the middle
```

Comment:
does the remaining need to change?

Reply:
there are two kinds of amazonaws url in china, both needs to be handled
- http://amazonaws.com.cn/
- http://www.amazonaws.cn/